corpus/filter.py

```diff
@@ -174,16 +174,18 @@ def run(self):
 
 
 class FilterCorpusBySegmentsJob(Job):
-    def __init__(self, corpus_file, segment_list, compressed=False, invert_match=False):
+    def __init__(
+        self, bliss_corpus, segment_file, compressed=False, invert_match=False
+    ):
         """
-        :param Path corpus_file:
-        :param Path segment_list:
+        :param Path bliss_corpus:
+        :param Path segment_file:
         :param bool compressed:
         :param bool invert_match:
         """
-        self.in_corpus = corpus_file
+        self.in_corpus = bliss_corpus
         self.segment_list = (
```

Comment:
`segment_list` implies to me a python-list of segment names. Maybe we should rename this to `self.segment_file_list` ?